pkg/webhook/mutation/pod/metadata/workload.go

```diff
@@ -102,6 +102,10 @@ func findRootOwner(ctx context.Context, clt client.Client, partialObjectMetadata
 		}
 	}
 
+	if partialObjectMetadata.ObjectMeta.Name == "" {
+		log.Info("podName is empty", "objectMeta.name", partialObjectMetadata.ObjectMeta.Name)
```

Comment:
you are saying something is empty, then printing it, why? 😅

Reply:
It was a long day... 🥲  thank you for pointing out :+1:

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/3170/commits/92c10486b2ce3fa35e54b105a364f1dff9bd273c